ssc/cmod_trough_physical.cpp

```diff
@@ -1238,13 +1266,13 @@ class cm_trough_physical : public compute_module
         if ((int)count != n_steps_fixed)
             throw exec_error("trough_physical", "The number of fixed steps does not match the length of output data arrays3");
 
-        ssc_number_t *p_m_dot_tes_dc = as_array("m_dot_tes_dc", &count);
-        if ((int)count != n_steps_fixed)
-            throw exec_error("trough_physical", "The number of fixed steps for 'm_dot_tes_dc' does not match the length of output data arrays");
-
-        ssc_number_t *p_m_dot_tes_ch = as_array("m_dot_tes_ch", &count);
-        if ((int)count != n_steps_fixed)
-            throw exec_error("trough_physical", "The number of fixed steps for 'm_dot_tes_ch' does not match the length of output data arrays");
+        //ssc_number_t *p_m_dot_tes_dc = as_array("m_dot_tes_dc", &count);
+        //if ((int)count != n_steps_fixed)
+        //    throw exec_error("trough_physical", "The number of fixed steps for 'm_dot_tes_dc' does not match the length of output data arrays");
+        //
+        //ssc_number_t *p_m_dot_tes_ch = as_array("m_dot_tes_ch", &count);
+        //if ((int)count != n_steps_fixed)
+        //    throw exec_error("trough_physical", "The number of fixed steps for 'm_dot_tes_ch' does not match the length of output data arrays");
```

Comment:
Prefer to keep old code around until people have exercised the new controller for a while

Reply:
I don't think anyone is going to go back through and clean up the commented code, especially with our maintenance practices. Other people wont' have the courage to delete it as they may think it's there for a reason. It'll just become more noise. Coding guidelines also recommend not doing this. It'll always be there in the source control so it can be easily found and restored if needed.